src/style.css

```diff
@@ -0,0 +1,123 @@
+* {
+  margin: 0;
+  padding: 0;
+  background-color: #f6f6f6;
+  color: #494949;
+}
+
+.to-do {
+  max-width: 400px;
+  margin: auto;
+  box-shadow: 5px 5px 20px #d8d8d8;
+  margin-top: 50px;
+}
+
+.today-todo,
+.text,
+.listItem,
+.clear,
+.newValue {
+  border: 1px solid #f6f6f6;
+  background-color: white;
+  width: 400px;
+  height: 50px;
+  display: flex;
+  align-items: center;
+  padding-left: 10px;
+}
+
+.text input {
+  font-style: italic;
+  width: 90%;
+  padding-left: 10px;
+}
+
+.submit,
+.text input {
+  background-color: white;
+  border: none;
+  height: 50px;
+}
+
+.text input:focus,
+.newInput {
+  outline: none !important;
```

Comment:
I think this line of code works fine without !important. Please remove it.

Reply:
:100: Done